README.md

```diff
@@ -1,3 +1,7 @@
 # Session notebook neuefische
 
 here i take notes during the web dev bootcamp and learn about markdown and git
+
+## new headline for experimenting with branches and pull requests
+
+So Lüder und Birk, mal gucken ob ihr euch das hier wirklich anguckt.
```

Comment:
Great work Lennart! Yes I checked your code, looks good to me ;)

Reply:
Thanks!